libdnf/advisory/advisory_module.cpp

```diff
@@ -28,20 +28,11 @@ namespace libdnf::advisory {
 // AdvisoryModule
 AdvisoryModule::AdvisoryModule(AdvisoryModule::Impl * private_module) : p_impl(private_module) {}
 
-AdvisoryModule::AdvisoryModule(const AdvisoryModule & src) : p_impl(new Impl(*src.p_impl)) {}
+AdvisoryModule::AdvisoryModule(const AdvisoryModule & src) = default;
 
 AdvisoryModule::AdvisoryModule(AdvisoryModule && src) noexcept = default;
 
-AdvisoryModule & AdvisoryModule::operator=(const AdvisoryModule & src) {
-    if (p_impl != src.p_impl) {
-        if (p_impl) {
-            *p_impl = *src.p_impl;
-        } else {
-            p_impl.reset(new Impl(*src.p_impl));
-        }
-    }
-    return *this;
-}
+AdvisoryModule & AdvisoryModule::operator=(const AdvisoryModule & src) = default;
 
 AdvisoryModule & AdvisoryModule::operator=(AdvisoryModule && src) noexcept = default;
```

Comment:
Could we squash all the 5 definitions into a single block with no newlines (unless clang-format complains I guess), here as well as in the header, and make it a standard unified way we do this, as it will be the same (except sometimes without the copy variants) in all classes that have pimpl?